Misc Geo/Merseyside Coastline.txt

```diff
@@ -6883,4 +6883,4 @@ N053.57.38.812 W010.07.14.612 N053.58.06.264 W010.07.20.953 32896
 N053.57.38.812 W010.11.20.682 N053.57.37.755 W010.11.48.138 32896
 N053.57.40.927 W009.50.30.287 N053.57.51.474 W009.50.27.118 32896
 N053.57.40.927 W009.50.48.239 N053.57.40.927 W009.50.30.287 32896
-N053.57.40.927 W009.51.12.533 N053.57.40.927 W009.51.17.810 32896
+N053.57.40.927 W009.51.12.533 N053.57.40.927 W009.51.17.810 32896
```

Comment:
Is `32896` Here a colour definition we should be creating?

Actually, thinking about it, do we actually use this file for anything since we have the entire UK coastline?

Reply:
I can only find it used in the Liverpool and Manchester ATM. Which is strange because it defines the west coast of Ireland as well.
![image](https://github.com/user-attachments/assets/1c24a1cc-5e64-421d-a5d0-023abd5ea333)
EGCC ATM
![image](https://github.com/user-attachments/assets/e56f060b-79c4-4eb3-a465-c8caad0be069)

Reply:
Maybe one for a separate issue to sort then, it seems a very wide slice of coastline, and a ~7000 line file coule probably be significantly reduced.

Reply:
Yep. Probably only need the west coast. Not any of Ireland / Holland. I'll create the issue and fix it some point next week.

Reply:
I've popped issue #5373 in, to keep track of this there as it's a different thing. No rush to work on it, but just thought it good to be tracking it.